include/roboteam_ai/control/controllers/BaseController.h

```diff
@@ -0,0 +1,25 @@
+//
+// Created by ratoone on 25-09-19.
+//
+
+#ifndef RTT_BASECONTROLLER_H
+#define RTT_BASECONTROLLER_H
+
+template <typename Input, typename Output>
+/**
+ * The base abstract class for the controllers. Any control method should inherit from this
```

Comment:
What is a controller and control method? This is still unclear for me when I read the documentation.

Reply:
I don't think we need this much detail here, everyone in the control team should know what a controller is